### ISSUE TITLE
Forms doesn't appear to work at all

### ISSUE THREAD

**Issue by zstarkebaumcalico** [xamarin/SignaturePad#39]
Can anyone confirm signature pad working with Xamarin Forms 2.2 or 2.3?

**Comment by eamreyes** [comment]
It didn't work for me on Android until I reset the stream.
`var image = await padView.GetImageStreamAsync(SignatureImageFormat.Png);`
`image.Seek(0, SeekOrigin.Begin);`
`MyFormsImage.Source = ImageSource.FromStream(() => image);`

 Afterwards it worked on Android with XF 2.2.031 and 2.3.2.127.

Tested on iOS and the stream that is returned from `GetImageStreamAsync` is at the start of the stream.

On Android I only got a black box when using `SignatureImageFormat.Jpg` and calling `ImageSource.FromStream`.

**Comment by gmeredith** [comment]
I have SignaturePad working great on Android. In iOS I get it to work in simulator, not on device.
Using VS 2015 Update 3, Xamarin.Forms 2.3.1.114.

**Comment by RCWade** [comment]
Using VS2015, I am also having trouble with iOS. 

My simulator tests and Debug direct to a device works fine. However, when I upload an IPA to Testflight, the build shows an empty space where the signature form should be.

In my Linker Options, the TestFlight build only works when I choose 'Don't Link', otherwise the app crashes on the splash screen.

**Comment by ckimes89** [comment]
If the signature pad is not loading on your iOS 10 device, I have found a workaround. The problem is with the linker (and presumably the linker ignoring the `Preserve` attribute). Add this code somewhere in your project, I'd recommend in the `AppDelegate.FinishedLaunching` method:

```
var renderer = new SignaturePadRenderer();
var view = new SignaturePadView();
var otherView = new SignaturePad.Forms.SignaturePadView();
```

**Comment by mattleibow** [comment]
The issue with the Android platform has been fixed in v1.5. 

The issue with iOS is an linker issue as mentioned. This will no longer be an problem with v2.0 (as yet unreleased).

This will work to prevent the linker form linking out the renderers:

```csharp
static bool falseflag = false;
static AppDelegate ()
{
    if (falseflag) {
        var ignore = new SignaturePad.Forms.iOS.SignaturePadRenderer ();
    }
}
```
